script.js

```diff
@@ -3,6 +3,9 @@ const pokeImg = document.querySelector('.pokeImg');
 const infoList = document.querySelector('.infoList');
 const pokeNameParagraph = document.querySelector('.pokeNameParagraph');
 const displayDateAndTimeParagraph = document.querySelector('.displayDateAndTimeParagraph');
+const input = document.querySelector('.input');
+
+let pokemonList = [];
 
 const displayDateAndTime = () => {
```

Comment:
I would avoid blocks of code that big - I believe You should place some new lines to separate logic blocks inside that function (i.e. between ln 52 and 53).
Ideally, every "logic block" should be a separate function ;)

Reply:
Done! :)